frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.test.ts

```diff
@@ -279,8 +280,8 @@ describe('sessionRecordingDataLogic', () => {
             ]
 
         it('no next url', async () => {
-            await expectLogic(logic, () => {
-                logic.actions.loadRecordingSnapshots()
+            await expectLogic(logic, async () => {
+                await logic.actions.loadRecordingSnapshots()
```

Comment:
nit: Don't think we need an async here?

Reply:
Ah good catch - added these in my battle against the test errors 🙈